docs/features/lists.md

```diff
@@ -11,15 +11,49 @@ keywords:
   - "Contributor Lists Management"
 ---
 
-The Lists feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Lists feature, gain granular insights into each contributor's activity and contributions.
+The OpenSauced Lists feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Lists feature, gain granular insights into each contributor's activity and contributions.
 
 Monitor individual commit histories, track who is actively engaged, identify alumni or new contributors, and even compare the performance of contributors against each other. This encompasses a wide range of metrics such as code commits, created and reviewed Pull Requests (PRs), as well as issue creation and commenting.
 
+Contributors can use Lists feature to track their own activity, while maintainers can use the feature to manage the community.
+
+## Lists Features for Contributors and Maintainers
+_For Contributors_
+
+- **Track Your Impact**: Monitor your commits and pull requests (PRs) across diverse projects, gaining a clear picture of your contributions.
+
+- **Benchmark Performance**: Compare your activity to others based on metrics like commits, created/reviewed PRs, and issue creation/commenting.
+
+- **Identify Peers & Alumni**: See who's actively involved, discover new contributors, and reconnect with returning members.
+
+- **Dive Deeper into Projects**: Explore contributors' repositories and top programming languages, gaining valuable insights into project landscapes.
+
+- **Understand Time Zones**: Collaborate more effectively with awareness of contributor time zones for streamlined communication.
+
+_For Maintainers_
+
+- **Monitor Project Health**: Track individual commit histories and gain an overall view of contributor activity and engagement.
+
+- **Identify Key Players**: Easily recognize new contributors, active members, and alumni, enabling you to nurture a thriving community.
+
+- **Foster Collaboration**: Facilitate smoother collaboration by understanding contributor activity and preferred communication times.
+
+- **Analyze Trends & Impact**: Gain valuable insights into project trends and contributor performance through detailed metrics analysis.
+
+- **Optimize Resource Allocation**: Identify areas where resources are needed based on contributor activity and skills.
+
+
 ![lists-page](../../static/img/lists-page.svg)
 
-## How to Create a List
+## Why Create Lists for Contributors and Maintainers?
```

Comment:
For this section, use what's contained in the [Why Create a List section of the guide](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#why-create-a-list) for maintainers and [What You Can do With Lists](https://docs.opensauced.pizza/contributors/contributors-guide-to-open-sauced/#what-you-can-do-with-lists) for Contributors.